pkg/transformer/openshift/openshift.go

```diff
@@ -135,6 +135,24 @@ func getAbsBuildContext(context string, composeFileDir string) (string, error) {
 		return "", err
 	}
 	prefix := strings.Trim(string(out), "\n")
+	index := strings.Index(context, prefix)
+	if index != -1 {
+		if prefix == "" {
```

Comment:
Okay, then it should be `index != -1 && prefix == ""` then an `if then` after for the `return context[index:], nil`